lib/fuzzy-finder-view.coffee

```diff
@@ -33,7 +34,29 @@ class FuzzyFinderView extends SelectListView
     @panel?.destroy()
 
   viewForItem: ({filePath, projectRelativePath}) ->
+
+    # Style matched characters in search results
+    filterQuery = @getFilterQuery()
+    matches = match(projectRelativePath, filterQuery)
+
     $$ ->
+
+      highlighter = (path, matches, offsetIndex) =>
+        lastIndex = 0
+        for matchIndex in matches
+          matchIndex -= offsetIndex
+          continue if matchIndex < 0 # If marking up the basename, omit path matches
+          unmatched = path.substring(lastIndex, matchIndex)
+          matchedChar = path[matchIndex]
+
+          @text unmatched if unmatched
+          @strong matchedChar, class: 'matched-char'
```

Comment:
I think this should be a `span` instead – this would allow for finer grained control over styling. (Bold / strong text isn't necessarily very visible in every theme.) Could you also add a style for this class in `styles/`?